### ISSUE TITLE
How to setup private and public port forwarding

### ISSUE THREAD

**Issue by LeeChanWu** [jitsi/jitsi-meet#3415]
Hello,
I am a new fresher in jitsi-meet server.
I installed jitsi-meet server in my own server by following quick install guide.
Server is ubuntu 16.04.

Problem1 : ICE failed and video call disconnected after 20seconds and reconnecting.

Logger.js:125 [modules/xmpp/JingleSessionPC.js] <r.peerconnection.oniceconnectionstatechange>:  (TIME) ICE failed P2P? false:	 103632.10000000254
Logger.js:125 [conference.js] <e.value>:  CONFERENCE FAILED: conference.setup_failed Error: ICE fail
    at r.peerconnection.oniceconnectionstatechange (JingleSessionPC.js:472)
    at RTCPeerConnection.peerconnection.oniceconnectionstatechange (TraceablePeerConnection.js:261)

I understood that I must add advanced configuration because my server is running behind NAT.
So, I tried https://github.com/jitsi/jitsi-meet/blob/master/doc/quick-install.md#advanced-configuration.

Problem2 : How to setup correctly?

/etc/jitsi/videobridge/sip-communicator.properties

org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.www.test.com/.*
org.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS=<Public.IP.Address>
org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=127.0.0.1
org.ice4j.ice.harvest.TCP_HARVESTER_PORT=4443
org.ice4j.ice.harvest.TCP_HARVESTER_MAPPED_PORT=4443

What is my mistake?
Regards

**Comment by damencho** [comment]
The value of NAT_HARVESTER_PUBLIC_ADDRESS should be the public address where clients over the internet can reach your jvb/jitsi-meet deployment.
`NAT_HARVESTER_LOCAL_ADDRESS=127.0.0.1` is not correct, this should be the private address where jitsi-videobridge is running. 
You also need to make sure the ports described in the quick-install manual are not closed by any firewall and to make sure that tour NAT device/router or whatever holds the public address is forwarding the ports to the machine in the NAT-ed network.

**Comment by damencho** [comment]
Please in the future when you have questions or problems, before opening an issue, discuss them first on the community forum: https://community.jitsi.org as stated in the issues template, thank you.

**Comment by LeeChanWu** [comment]
Hello, @damencho 
Thank you for your reply.
It was my mistake.
My server was not behind NAT.
When I open 4443 port in firewall, problem was fixed.
Thank you very much

I have another help from you.
I integrated android jitsi with pre-build SDK artifacts/binaries.
When I loadURL() in JitsiMeetView, app was crashed.
I created with android native project, not react-native project.
I have to create react-native project and integrate?
Regards
